<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageNotifyWidget</class>
 <widget class="QWidget" name="MessageNotifyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>24</y>
     <width>300</width>
     <height>176</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_titleBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>300</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>268</x>
     <y>0</y>
     <width>24</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_logo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>2</y>
     <width>22</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_lname">
   <property name="geometry">
    <rect>
     <x>52</x>
     <y>3</y>
     <width>120</width>
     <height>18</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei UI Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>XWCWALLET</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
